Sources/EdgeNetworkHandlers/NetworkResponseHandler.swift

```diff
@@ -188,7 +188,7 @@ class NetworkResponseHandler {
         Log.trace(label: LOG_TAG, "dispatchEventErrors - Processing \(unwrappedErrors.count) errors(s) for request id: \(requestId)")
         for error in unwrappedErrors {
 
-            if let errorAsDictionary = try? error.asDictionary() {
+            if let errorAsDictionary = error.asDictionary() {
```

Comment:
For me, Xcode is flagging this line as not needing the `try` statement, probably because it's using `AnyCodeable.asDictionary` though I would expect `EdgeEventEncodable.asDictionary` to be used. 
I think we should update `EdgeEventEncodable.asDictionary` to remove the throws clause and just return `nil` on error, same as `AnyCodable.asDictionary`.